tests/nightly/test_large_array.py

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+from mxnet import gluon, nd
+
+def test_ndarray2numpy():
+    m = gluon.nn.Embedding(14000, 128)
+    m.initialize()
+    ind = nd.zeros((700000, 128))
+    x = m(ind)
+    x.shape
+    test=x.asnumpy()
+    assert (x.shape == test.shape)
+
+if __name__ == '__main__':
+    test_ndarray2numpy()
```

Comment:
I think you need to update https://github.com/apache/incubator-mxnet/blob/master/tests/nightly/Jenkinsfile to enable nightly test @marcoabreu correct me if i'm wrong

Reply:
To be honest, I can't find any evidence of the fact that we run the python tests in tests/nightly at all right now :(

I'd rather put these tests in tests/python/unittest or gpu. The most preferred method on the other hand (considering edge devices) would be to make a C++ test and then mock the memory allocation. 

@larroy @lebeg might want to chime in

Reply:
@marcoabreu Do we have some integration test suite? I can add a C++ unit test but still prefer if we can test this from end-to-end.

Reply:
Sorry, I'm currently swamped with high priority tasks and I don't have time to review your task or assist with inqueries.

Reply:
I would suggest:
```python
import unittest
class TestLargeArray(unittest.TestCase):
    ...

if __name__ == '__main__':
    unittest.main()
```
And to add a empty line to the end of the file.

Reply:
Well, my suggestion is more about the python code style, where to put it is another topic. Event if it would be a nightly test, you still want to have it in a good shape.